Libraries/LibFileSystem/FileSystem.cpp

```diff
@@ -368,6 +394,7 @@ ErrorOr<off_t> size_from_fstat(int fd)
     return st.st_size;
 }
 
+#ifndef AK_OS_WINDOWS
 ErrorOr<off_t> block_device_size_from_ioctl(StringView path)
```

Comment:
I deleted some functions. 
The following functions are not used too, but I've kept them because they might be needed later:
```
copy_file 
copy_directory
copy_file_or_directory
move_file
can_delete_or_move
```